lib/active_model/serializer/associations.rb

```diff
@@ -12,9 +12,10 @@ module Associations
 
       DEFAULT_INCLUDE_TREE = ActiveModel::Serializer::IncludeTree.from_string('*')
 
-      included do |base|
-        class << base
-          attr_accessor :_reflections
+      included do
+        with_options instance_writer: false, instance_reader: true do |serializer|
```

Comment:
I don't know if there's a difference between the block variables `base` and `serializer` in what we want here.